### PR TITLE
Add extra buttons to colony header

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Some of the buttons have weird or different styles from the ones in the design. I have decided not to update those for now as I saw there were some discussions around consolidating them anyway.

To test this, just go to the colony dashboard and check that the new 2 extra buttons are appearing in the header, in the case that you already joined the colony, only 1 of them should appear.

FireShot Capture 175 - Extensions } {extensionId, select, VotingReputation {Governance} OneT_ - localhost
FireShot Capture 176 - Extensions } {extensionId, select, VotingReputation {Governance} OneT_ - localhost
FireShot Capture 177 - Extensions } {extensionId, select, VotingReputation {Governance} OneT_ - localhost
FireShot Capture 178 - Extensions } {extensionId, select, VotingReputation {Governance} OneT_ - localhost

### CODE REVIEW

File: src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx
Comment (thomassdk) [inline src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx:1]: The name of this file breaks webpack, it should be capitalised: `InviteMembersModal.tsx`

File: src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  },
+});
+
+// @TODO: Add logic to get the real amount of invites available and used.
+const invitesAvailable = 100;
+const invitesUsed = 0;
+
+const InviteMembersModal = ({ isOpen, onClose }: Props) => {
+  const { colony } = useColonyContext();
+  const inviteLink = `app.colony.io/${colony?.name}/invite`;
```
Comment (thomassdk) [inline src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx:50]: You can also get the correct invite code (which replaces the `invite` string in this path with https://github.com/JoinColony/colonyCDapp/pull/1207/commits/b4553e07139c0f8dbd12550f214c3e14b428beae

**Review by thomassdk — COMMENTED** [review]
Looks good some nice work here :+1: 

You can now get all the data that is missing if you base this PR on https://github.com/JoinColony/colonyCDapp/pull/1207

File: src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    id: `${displayName}.buttonText`,
+    defaultMessage: `{isCopied, select,
+      true {Link copied}
+      other {Copy link}
+    }`,
+  },
+});
+
+// @TODO: Add logic to get the real amount of invites available and used.
+const invitesAvailable = 100;
```
Comment (chmanie) [inline src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx:45]: Shall we hold out with this until this functionality is merged?

**Review by chmanie — APPROVED** [review]
Great work! We can either merge it now or wait until we have the actual invite functionality

File: src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx
Comment (ArmandoGraterol) [reply src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx:1]: For some reason I can't explain (I know Saint Peter won't call my name) I renamed this file and the change wasn't being recognized by git, should be good now.

File: src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx (hunk 2 above)
Comment (ArmandoGraterol) [reply src/components/v5/common/Modals/InviteMembersModal/inviteMembersModal.tsx]: Yup, I'll merge this PR as I'm a bit biased towards doing small PRs but as soon as that functionality is available I can come back in and make another PR to connect it.

**Pull request merged**